gradle/libs.versions.toml

```diff
@@ -11,7 +11,7 @@ grails-bom = { module = 'org.grails:grails-bom', version.ref = 'grails' }
 grails-core = { module = 'org.grails:grails-core' }
 greenmail = { module = 'com.icegreen:greenmail', version.ref = 'greenmail' }
 grails-docs = { module = 'org.grails:grails-docs' }
-grails-gsp = { module = 'org.grails.plugins:gsp' }
+grails-gsp = { module = 'org.grails:grails-gsp' }
```

Comment:
Does the taglib @AstTransformer TagLibraryTransformer still get applied to the taglib?

Reply:
Good catch! I have added `org.grails:grails-web-taglib` to fix that.